pkg/client/unversioned/request.go

```diff
@@ -718,12 +729,8 @@ func (r *Request) Do() Result {
 	return result
 }
 
-// DoRaw executes the request but does not process the response body.
+// DoRaw is a conveneience function, executing the request (without processing the response body).
```

Comment:
convenience

...actually revert the comment, this is not a convenience function, this is just how you do things.